elixir/apps/domain/lib/domain/flows/flow.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Domain.Flows.Flow do
+  use Domain, :schema
+
+  schema "flows" do
```

Comment:
This is a good start. We'll want to have src/dest ports too for these to be uniquely identifiable for audit logs, but that'll probably have to be reported by connlib on both sides of the tunnel. Flow size would be a nice addition too.